src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PhysicalDetailResourceTest.kt

```diff
@@ -0,0 +1,176 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.resource
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.web.reactive.function.BodyInserters
+import uk.gov.justice.digital.hmpps.prisonersearch.QueueIntegrationTest
+import uk.gov.justice.digital.hmpps.prisonersearch.model.RestResponsePage
+import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.PaginationRequest
+import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.PhysicalDetailRequest
+
+class PhysicalDetailResourceTest : QueueIntegrationTest() {
+  private companion object {
+    private var initialiseSearchData = true
+  }
+
+  @BeforeEach
+  fun setup() {
+    if (initialiseSearchData) {
+
+      setupIndexes()
+      indexPrisoners()
```

Comment:
Can this extend `AbstractSearchDataIntegrationTest` from the last PR?

Reply:
certainly, will do in next PR.  Raised them both at the same time and didn't want one of them conflicting with the other one